src/__tests__/actions/currentUserActions.test.js

```diff
@@ -134,18 +144,18 @@ describe('currentUserActions', () => {
       const expectedActions = [
         {
           type: SET_CURRENT_USER_FOLLOWING,
-          payload: true,
+          payload: true
         },
         {
           type: SET_USER_ACTION_FAILURE,
-          payload: 'User not found',
```

Comment:
I thought you guys were following the trailing commas system, what's happening here?

Reply:
We are still  following the trailing commas system, I started this branch over unmerged branch, I've just updated this file